libbpf-rs/Cargo.toml

```diff
@@ -26,9 +26,11 @@ nix = "0.22"
 num_enum = "0.5"
 strum_macros = "0.21"
 vsprintf = "2.0"
+lazy_static = "1.4"
 
 [dev-dependencies]
 libc = "0.2"
 plain = "0.2.3"
 scopeguard = "1.1"
 serial_test = "0.5"
+log = "0.4"
```

Comment:
Is this needed? I'm not seeing where log is being used in the code.

Reply:
This is used by the doctests for the example code with `set_print` which shows passing output to the log crate.